app/code/Magento/Sales/Test/Mftf/Test/AdminSubmitsOrderWithAndWithoutFieldsValidationTest.xml

```diff
@@ -33,8 +33,8 @@
         <!--<actionGroup ref="NavigateToNewOrderPageNewCustomerActionGroup" stepKey="navigateToNewOrderPage"/>-->
         <actionGroup ref="AdminOrdersPageOpenActionGroup" stepKey="navigateToOrderIndexPage"/>
         <see selector="{{AdminHeaderSection.pageTitle}}" userInput="Orders" stepKey="seeIndexPageTitle"/>
-        <click selector="{{AdminOrdersGridSection.createNewOrder}}" stepKey="clickCreateNewOrder"/>
-        <click selector="{{AdminOrderFormActionSection.CreateNewCustomer}}" stepKey="clickCreateCustomer"/>
+        <actionGroup ref="AdminClickCreateNewOrderOnOrderGridPageActionGroup" stepKey="clickCreateNewOrder"/>
+        <actionGroup ref="AdminClickCreateNewCustomerForOrderActionGroup" stepKey="clickCreateCustomer"/>
         <see selector="{{AdminHeaderSection.pageTitle}}" userInput="Create New Order" stepKey="seeNewOrderPageTitle"/>
```

Comment:
Could I kindly ask you to refactor this assertion throughout the test as well to make this PR even more valuable? 

Thanks

Reply:
@rogyar **Done**